community/kernel/src/main/java/org/neo4j/unsafe/impl/batchimport/input/Groups.java

```diff
@@ -66,4 +66,36 @@ public synchronized Group getOrCreate( String name )
         }
         return group;
     }
+
+    private IllegalStateException mixingOfGroupModesException()
+    {
+        return new IllegalStateException( "Mixing specified and unspecified group belongings " +
+                "in a single import isn't supported" );
+    }
+
+    public synchronized Group get( String name )
```

Comment:
It's a bit hard to follow the flow, but it looks like this method was introduced to help `verifyRelationshipHeader()` assert that the header is there. Wouldn't a `Groups.isPresent(String)` make more sense and eliminate the duplicate code?

Reply:
`isPresent` would be implemented almost like `get` wouldn't it?

Reply:
That is the case, yes, disregard my previous comment :)